### ISSUE TITLE
Path too long

### ISSUE THREAD

**Issue by james-radford** [dashton82/FakeScript#9]
I've received multiple errors because we're using npm at my place and I think windows/ the script is not able to handle the amount of sub folders.  I get System.Io.PathTooLongException error when running RunBuild in the command prompt.

**Comment by dashton82** [comment]
This is a known windows issue _sigh_

Easiest work around is to remove the Clean Directories from the dependency tree - this will leave you with the following:

```
"Set version number"
   ==>"Set Solution Name"
   ==>"Update Assembly Info Version Numbers"
  // ==>"Clean Directories" 
   ==>"Clean Projects"
   ==>"Build Projects"
   ==>"Run NUnit Tests"
   ==>"Build Cloud Projects"
   ==>"Build Database project"
   ==>"Build WebJob Project" 
   ==>"Publish Solution"  
   ==>"Compile Views"
   ==>"Create Nuget Package"
```

**Comment by dashton82** [comment]
Is there a __node_modules__ folder in your solution?

**Comment by dashton82** [comment]
Fixed as part of a1dffbb9dbd0f81ef4eca66c016d5b4f8f06c249

**Comment by james-radford** [comment]
ok thanks, I'm running the changed version as we speak, so far so good.
